pkg/util/v1alpha3/katibclient/katib_client.go

```diff
@@ -123,6 +124,14 @@ func (k *KatibClient) CreateExperiment(experiment *experimentsv1alpha3.Experimen
 	return nil
 }
 
+func (k *KatibClient) UpdateExperiment(experiment *experimentsv1alpha3.Experiment, namespace ...string) error {
```

Comment:
is this method used for UI? and it seems `namespace` param isn't used

Reply:
I am not sure about it. Shall we do this separately?

Reply:
If this method is not useful in your PR topic, I think we'd better remove it.

Reply:
No. this method is currently used in resume_e2e_experiment.go script in this PR to update the experiment.